types.cc

```diff
@@ -3455,6 +3455,16 @@ static std::string unescape(sstring_view s) {
     return result;
 }
 
+// Replace ":" with "\:" and "@" with "\@".
+static std::string escape(sstring_view s) {
+    static thread_local std::regex colon_re(":");
+    static thread_local std::regex at_re("@");
+    std::string result(s);
+    result = std::regex_replace(result, colon_re, "\\:");
+    result = std::regex_replace(result, at_re, "\\@");
+    return result;
```

Comment:
Can do a single pass with "[@:]" and "\\$1"

Reply:
Thanks. I've also changed the 'unescape' function.